packages/vscode-extension/package.json

```diff
@@ -40,37 +40,21 @@
     "out/**/*"
   ],
   "activationEvents": [
-    "*",
-    "onCommand:fx-extension.openWelcome",
-    "onCommand:fx-extension.openSamples",
-    "onCommand:fx-extension.openDocument",
-    "onCommand:fx-extension.cmpAccounts",
+    "onStartupFinished",
     "onCommand:fx-extension.create",
-    "onCommand:fx-extension.addCapability",
-    "onCommand:fx-extension.update",
-    "onCommand:fx-extension.openManifest",
-    "onCommand:fx-extension.validateManifest",
-    "onCommand:fx-extension.build",
-    "onCommand:fx-extension.provision",
-    "onCommand:fx-extension.deploy",
-    "onCommand:fx-extension.publish",
-    "onCommand:fx-extension.updateAad",
-    "onCommand:fx-extension.openAppManagement",
-    "onCommand:fx-extension.openBotManagement",
-    "onCommand:fx-extension.openReportIssues",
     "onCommand:fx-extension.getNewProjectPath",
-    "onCommand:fx-extension.openReadMe",
     "onCommand:fx-extension.openDeploymentTreeview",
+    "onCommand:fx-extension.openDocument",
```

Comment:
Why remove some of commands while keeping others?

Reply:
These commands are the ones in welcome view and walkthrough, so they are kept making sure extension will be activated to run them. The removed ones are used in teamsfx project, they will not show in command palette until extension is activated and detects that current project is teamsfx. User are not allowed to run these commands in a non-teamsfx project, thus they should not be used to activate extension.

Reply:
So before extension activate complete, nothing will happen if user execute these commands(like provision, deploy, ...) in command palette?

Reply:
Yes, actually user cannot execute these teamsfx commands (provision, etc.) from command palette because they depend on  context value which is set after activation is completed.